### DISCUSSION TITLE
`leia` com parâmetros a mais, deveria retornar erro?

### DISCUSSION THREAD

**Discussion by samuelrvg** [DesignLiquido/delegua#212] [P&R] [answered]
`var teste = leia('Digite algo:', 121123, 'abc')`

**Comment by leonelsanchesdasilva** [comment, marked as answer]
Qual o objetivo dos demais parâmetros? A meu ver, `leia` deveria ter apenas um parâmetro.

**Reply by samuelrvg** [reply]
Concordo com você, mas no caso de passar mais parâmetros, não deveria gerar algum erro ?
